pkg/filter/filter.go

```diff
@@ -2,17 +2,14 @@ package filter
 
 import (
 	"context"
-
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
+	"github.com/kubesphere/notification-manager/pkg/apis/v2beta2"
```

Comment:
Can we change like this to avoid prefix a API call with a version number:
```go
	nm "github.com/kubesphere/notification-manager/pkg/apis/v2beta2"
```
or

```go
	nmv2beta2 "github.com/kubesphere/notification-manager/pkg/apis/v2beta2"
```

Reply:
The versions of the api are divided into `v1beta1` and `v2beta2`, wouldn't it be confusing to use `nm` instead of `v2beta2`?

Reply:
Merge this for now